pkgs/applications/emulators/dolphin-emu/primehack.nix

```diff
@@ -146,6 +146,6 @@ stdenv.mkDerivation rec {
     # x86_32 is an unsupported platform.
```

Comment:
```suggestion
```
i686 is barely supported.

Reply:
I'll do this to regular Dolphin too, then.